static_src/components/panel_row.jsx

```diff
@@ -29,9 +27,10 @@ export default class PanelRow extends React.Component {
   render() {
     const props = this.props;
     const styleClass = props.styleClass && `panel-row-${this.props.styleClass}`;
+    const mainClass = props.styleClass !== 'boxed' && 'panel-row';
 
     return (
-      <div className={ this.styler('panel-row', styleClass) }>
+      <div className={ this.styler(mainClass, styleClass) }>
```

Comment:
I liked the way it was working before, using css base and a css modifier class. Why the need for this change?